Package.swift

```diff
@@ -48,7 +49,7 @@ func resolveTargets() -> [Target] {
 let package = Package(
         name: "Purchases",
         platforms: [
-            .macOS(.v10_12), .iOS(.v9)
+            .macOS(.v10_12), .iOS(.v9), .watchOS("6.2"), .tvOS("13.4")
```

Comment:
We have some tvOS apps already, will this change affect them?

Reply:
I'm updating this one per the conversation we had a while back, I'd forgot to perform the update here.

Reply:
I'm not sure they were able to use Purchases through SPM though - note that the TV platform wasn't defined for it before this PR